README.md

```diff
@@ -1,5 +1,8 @@
 # FLEA MARKET BACK-END
 
+## DB schema:
+
+
```

Comment:
need to place the image in root of repository (the same level as README.md), add the image to index (git)

then you can write here:
```markdown
![DB schema](/schema.png)
```
**"DB schema"** – description of picture (shows when you place your cursor on the image)
**"/schema.png"** – path to image. As your image is located in root of repo, path should be "/$nameOfYourImageWithExtension"

after commiting that stuff there should be a picture in README.md
you can also check it in IDEA
![image](https://github.com/safgbad/flea-market/assets/79574467/9f383e5e-d07a-4a18-a272-18958ddbb8b6)
there should be the image in preview